### PR TITLE
feat(range): make use of design tokens

### PR DESCRIPTION
**Pull request by mivaylo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by mivaylo** [comment]
CDE-694

File: projects/angular/src/forms/styles/_range.clarity.scss [hunk 1]
```diff
@@ -32,61 +42,117 @@
         forms-variables.$clr-forms-range-progress-fill-color,
         variables.$clr-use-custom-properties
       );
+      @include forms-mixins.range-border-style(
```
[the remark this answers is not in the document]
Comment (mivaylo) [reply projects/angular/src/forms/styles/_range.clarity.scss:45]: Well that would simplify things, but I don't understand the figma this way. I understand we want the flexibility of having 3 different colors (background, progress and border) two of which are equal by default, but not always:

image

Maybe I am wrong though

[the remark this answers is not in the document]
Comment (Jinnie) [reply projects/angular/src/forms/styles/_range.clarity.scss:45]: I am just talking about the border of the progress. Which is white in the current implementation and gives the strange overlap effect on the dot.

File: projects/angular/src/forms/range/range-container.ts
Comment (dtsanevmw) [inline projects/angular/src/forms/range/range-container.ts:1]: Do we really need a functional changes  just for the token migration?

**Review by dtsanevmw — COMMENTED** [review]
I agree that we have a "border" around the non-filled part of the range but I see that in core that's not available. So maybe it will be better to stick with migrating what core have and leave the new visual things which are more of a rework for the future release.

File: projects/angular/src/forms/range/range-container.ts
Comment (mivaylo) [reply projects/angular/src/forms/range/range-container.ts:1]: I am not sure this change qualifies as a functional one, but it is required from the new design (we now want border-radius)

This is the behavior of the .fill-input blue part without the fix:

https://github.com/vmware-clarity/ng-clarity/assets/132376042/0535ddba-8ddb-421d-b44c-1f1d5c255e65

and this is the behavior with the fix:

https://github.com/vmware-clarity/ng-clarity/assets/132376042/5808876a-a9a8-4e7a-b07c-9aa3909e17d0


so I don't know whether we need it but I think we better have it

**Comment by mivaylo** [comment]
Okay, I will revert the logic in the range-container file

File: projects/angular/src/forms/styles/_mixins.forms.scss [hunk 2]
```diff
@@ -205,17 +205,33 @@
...
   -webkit-appearance: none;
   left: 0;
-  height: $height;
 
+  @include mixins.css-var(height, $height-prop, $height, $use-custom-props);
   @include mixins.css-var(background-color, $background-color-prop, $background-color, $use-custom-props);
 }
 
+@mixin range-border-style(
+  $border-color,
```
Comment (mindarelus) [inline projects/angular/src/forms/styles/_mixins.forms.scss:221]: maybe rename to border-color-fallback, as well as the other fallbacks, as it's misleading

**Review by Jinnie — APPROVED** [review]
LGTM

**Pull request merged**